src/api/index.js

```diff
@@ -9,18 +9,32 @@ import getSecret from '../utils/rds/getSecret.js';
 import deleteSecret from '../utils/rds/deleteSecret.js';
 import updateSecret from '../utils/rds/updateSecret.js';
 import addSecret from '../utils/rds/addSecret.js';
+import client from '../utils/rds/dbClient.js';
+import Secrets from '../utils/rds/model.js';
 
 const app = express();
 
 app.use(cors());
 app.use(express.json());
 app.use(express.urlencoded({ extended: true }));
 
+app.use((req, res, next) => {
+    console.log('req.headers is: ', req.headers);
```

Comment:
This should be deleted before production.

Reply:
I'll make a note of that for next commit.